src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -647,8 +647,11 @@ export default {
       if (header && alteration && business) {
         const newLegalType = this.getCorpTypeDescription(alteration.business?.legalType)
         const oldLegalType = this.getCorpTypeDescription(business.legalType)
-        const courtOrderNumber = '' // FUTURE
-        const isArrangement = false // FUTURE
+        const courtOrder = alteration?.courtOrder
```

Comment:
Conditional chaining is always a red flag for me when there is no fallback handling. I ask myself, what does it mean when the object is null? What happens next?

In this case, I see that you check that `alteration` is truthy on line 647, so actually this conditional chaining is not needed and this code is safe.

Line 651 below is a great example of conditional chaining with a fallback 👍

Reply:
updated line 650 to stop using conditional chaining